### PR TITLE
Unify resource filenames across the repository

### PR DESCRIPTION
**Pull request by DHowett-MSFT**
This unifies the rest of the projects around the resource structure laid out in WindowsTerminalUniversal. Now we'll have a single flat structure for resource files and keep the qualifiers in their filenames. It's easier to manage this way.

### CODE REVIEW

**Comment by DHowett-MSFT** [comment]
This also pulls our application name out into the language-free resource in CascadiaPackage: this might help with #2289.

**Comment by DHowett-MSFT** [comment]
> 
> 
> This also pulls our application name out into the language-free resource in CascadiaPackage: this might help with #2289.

[narrator voice] it did not

File: src/cascadia/CascadiaPackage/Resources/Resources.language-en.resw [hunk 1]
```diff
@@ -120,19 +120,7 @@
...
-  <data name="AppNameDev" xml:space="preserve">
-    <value>Windows Terminal (Dev Build)</value>
-  </data>
-  <data name="AppShortName" xml:space="preserve">
-    <value>Terminal</value>
-  </data>
-  <data name="AppShortNameDev" xml:space="preserve">
-    <value>Terminal (Dev)</value>
-  </data>
-</root>
```
Comment (carlos-zamora) [inline src/cascadia/CascadiaPackage/Resources/Resources.language-en.resw]: Where'd this move to?

Comment (DHowett-MSFT) [reply src/cascadia/CascadiaPackage/Resources/Resources.language-en.resw]: Resources.resw.

**Pull request merged**